### DISCUSSION TITLE
Localization issue

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#16510] [Q&A]
I have a project based on OrchardCore 1.8.3, where:
- I have enabled the modules `OrchardCore.Localization` and `OrchardCore.ContentLocalization.ContentCulturePicker`
- I have set the cultures to English (en) and Italian (it)
After setting the Italian culture through the Content Culture Picker, when I type `localhost:8080` or `localhost:8080/`, the application always redirects to the home page in **English (en)** instead of the **Italian (it)** one.

I haven't set any CustomRequestCultureProvider in the Startup.cs file.

Could anyone help me with this issue?
Thank you for attention.

**Comment by hishamco** [comment]
> I haven't set any CustomRequestCultureProvider in the Startup.cs file.

No need, once you enable the content culture picker there's a culture provider that should handle the localization behind the scene
